lib/lumberjack/client.rb

```diff
@@ -39,7 +41,7 @@ def connect
 
     public
     def write(elements)
-      @socket.write_sync(elements)
+      @socket.write_sync(elements, :json => @opts[:json])
```

Comment:
both is possible. default is set when creating socket/client + can be overwritten (optionally) in write/write_sync via option hash